### PR TITLE
Add option to take dog sled adventure, add new page

### PR DESCRIPTION
**Pull request by karashirl**
Please review this, @casey-collab

### CODE REVIEW

File: english/dog-sled/dogsled.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+You embark on a dog sled adventure across the globe, setting the record for the fastest time. Then you realize you're
+[right back where you started](english/marshmallow.md).
```
Comment (casey-collab) [inline english/dog-sled/dogsled.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by karashirl** [comment]
Fixed the link

**Comment by karashirl** [comment]
Fixed the link @casey-collab

**Comment by casey-collab** [comment]
Looks great! :gem:

**Pull request merged**